gwpy/cli/timeseries.py

```diff
@@ -109,3 +135,14 @@ def scale_axes_from_data(self):
         ymax = max(ts.value.max() for ts in cropped)
         self.plot.gca().yaxis.set_data_interval(ymin, ymax, ignore=True)
         self.plot.gca().autoscale_view(scalex=False)
+
+    @classmethod
+    def arg_channels(self, parser):
+        """Add an option to overlay different times rather than on the
+        same time scale"""
+        super().arg_channels(parser)
+        group = parser.add_argument_group(
+                'Data options time series', 'Allow overlay')
+        group.add_argument('--dt', action='store_true',
```

Comment:
@areeda, my comment here doesn't seem to have been added, so I'll repost it:

Can you rename this flag? `--dt` gives the user the impression that they are changing the `TimeSeries.dt` attribute, i.e. that they're resampling the timeseries. Maybe something like `--start-time-offset`?

Reply:
I will rename to something to distinguish it from the TimeSeries property dt

Reply:
Start time offset is too close to what the epoch option does. I think dt is short and clear but if you insist how  about --overlay-times?

Reply:
That makes sense to me, `--overlay-times` sounds sensible